C/plugins/north/OMF/include/omflinkeddata.h

```diff
@@ -0,0 +1,104 @@
+#ifndef OMFLINKEDDATA_H
+#define OMFLINKEDDATA_H
+/*
+ * Fledge OSIsoft OMF interface to PI Server.
+ *
+ * Copyright (c) 2022 Dianomic Systems
+ *
+ * Released under the Apache 2.0 Licence
+ *
+ * Author: Mark Riddoch
+ */
+
+#include <map>
+#include <reading.h>
+#include <OMFHint.h>
+
+/**
+ * The OMFLinkedData class.
+ * A reading is formatted with OMF specifications using the linked
+ * type creation scheme supported for OMF Version 1.2 onwards.
+ *
+ * This is based on the new mechanism discussed at AvevaWorld 2022 and
+ * the mechanism is detail in the Google Doc,
+ * https://docs.google.com/document/d/1w0e7VRqX7xzc0lEBLq-sYhgaHE0ABasOa6EC9dJMrMs/edit
+ *
+ * The principle is to use links to contianers in OMF with each contianer beign a single
```

Comment:
Typos in this paragraph: contianer -> container (2x), beign -> being, tyoes -> types, vis -> via.